### ISSUE TITLE
Dashboards for employees with little or no data may fail to render

### ISSUE THREAD

**Issue by mskyttner** [KTH-Library/bibliomatrix#46]
It would be great if rendering of abm.Rmd goes ok regardless of kthid.

I think it currently fails on some kthids, try for example this kthid: u1o2ujjd

It looks like it is this chunk in abm.Rmd that chokes:

```r
DT::datatable(df_diva %>% rename("WoS coverage" = "WoS_coverage",
                                 "DiVA publication type" = "Publication_Type_DiVA",
                                 "Total" = "P_frac"),
              rownames = FALSE,
              extensions = "Buttons",
              options = list(
                bPaginate = FALSE,
                pageLength = 100,
                dom = 'Bt',
                buttons = c("copy", "csv", "excel"),
                title = "ABM_publications")
              ) %>%
  formatRound(2:9, digits = 1, mark = "") %>%
  formatPercentage("WoS coverage", digits = 1) %>%
  abm_format_rows()
```

The message is `Unknown column P_frac`.

**Comment by agnepagne** [comment]
Fixed in 2885a055d6c9e115dd7731275a0678abd47f43ac I belive. Does it still fail with latest code, @mskyttner?

**Comment by mskyttner** [comment]
No it works! Thanks @agnepagne Closing this.
